keps/sig-cli/20191111-kui.md

```diff
@@ -24,7 +25,7 @@ superseded-by:
 
 ---
 
-# Kubectl Graphical Plugins
+# KEP-20191111 Kubectl Graphical Plugins
```

Comment:
The new template is actually different:
- it requires opening tracking issue in this repo (see https://github.com/kubernetes/enhancements/issues/2254 as an example)
- it requires creating a directory in keps/sig-cli/<that-issue-number>-<kep-name>
- moving the KEP to README.md file in that dir
- extracting metadata to kep.yaml

See https://github.com/kubernetes/enhancements/tree/master/keps/NNNN-kep-template for the template.

Reply:
Thanks - I'll take a look.